app/components/bordered_cards_component.rb

```diff
@@ -2,7 +2,16 @@
 
 class BorderedCardsComponent < ViewComponent::Base
   def initialize(cards:, class_name: nil, cards_per_row: 2)
-    @cards = cards
+    # initially this component only supported one link per card, but now we need
+    # to add mutiple links per card. old callers may be using cards with a :link
+    # field, so we copy that over to a :links field with one one element
+    @cards = cards.map! do |card|
+      card[:links] ||= []
+      card[:links] << card[:link] if card.key?(:link)
+      card.delete(:link)
+      card
+    end
+
```

Comment:
Could we decide whether we want to be a map or an each please 😆 

```suggestion
    @cards = cards

    @cards.each do |card|
      card[:links] ||= []
      card[:links] << card[:link] if card.key?(:link)
      card.delete(:link)
    end
```